Packs/CommonScripts/Scripts/Jmespath/jmespath_transformer.py

```diff
@@ -0,0 +1,24 @@
+import demistomock as demisto
+from CommonServerPython import *
+import jmespath
+
+
+def jmespath_search(expression: str, value: dict) -> dict:
+    try:
+        expression_compiled = jmespath.compile(expression)
+    except Exception as err:
+        return_error(f"Invalid expression - {err}")
+    result = expression_compiled.search(value)
+    return result
+
+
+def main():
+    args = demisto.args()
+    value = args.get("value")
+    expression = args.get("expression")
+    result = jmespath_search(expression, value)
+    demisto.results(result)
```

Comment:
Please use return_results instead of demisto.results()
https://xsoar.pan.dev/docs/integrations/code-conventions#return_results

Reply:
Done.